sql/4_2_2-to-4_3_1_upgrade.sql

```diff
@@ -181,3 +181,21 @@ CREATE TABLE `valueset` (
 -- updating nqf code for cqm measure blood pressure
 UPDATE `clinical_rules` set `cqm_nqf_code` = '0018' where `id` = 'rule_htn_bp_measure_cqm';
 --
+
+
+#IfMissingColumn layout_options validation
+ALTER TABLE layout_options ADD COLUMN validation varchar(100) default NULL;
+#EndIf
+#IfNotRow list_options option_id LBF_Validations
```

Comment:
My comment is not about the ALTER TABLE.  It is to suggest changing this line:
#IfNotRow list_options option_id LBF_Validations